.github/workflows/diff.yml

```diff
@@ -31,10 +41,19 @@ jobs:
     runs-on: ubuntu-latest
     container: obolibrary/odklite:v1.3.0
     steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: '#gogoeditdiff'
```

Comment:
there could be a different trigger for the reasoned version to safe resources for larger ontologies?

Reply:
if we want a different trigger, i would suggest two separate actions, we can split it out